src/test/resources/db/migration/V1__init.sql

```diff
@@ -1,24 +1,22 @@
-create table hibernate_sequence (
-    next_val bigint
-);
+create schema if not exists bookcrossing;
 
-insert into hibernate_sequence values ( 1 );
+create sequence bookcrossing.hibernate_sequence start 1 increment 1;
 
-create table t_action_mail_user (
+create table if not exists bookcrossing.t_action_mail_user (
     confirmation_mail varchar(255) not null,
     type integer,
     user_id integer not null,
     primary key (confirmation_mail)
 );
 
-create table t_attach (
+create table if not exists bookcrossing.t_attach (
     attach_id integer not null,
-    data longblob,
+    data oid,
```

Comment:
Why did you decide to change column type?
`oid` in PG is numeric object identifier https://www.postgresql.org/docs/current/datatype-oid.html
`longblob` in MySQL is a binary large object https://tableplus.com/blog/2019/10/tinyblob-blob-mediumblob-longblob.html

Reply:
Since pg by default stores large files in pg_largeobject and returns their oid

Reply:
annotation @Lob indicated that the data would be large, so it was saved as large_object